app/src/main/java/org/dhis2/usescases/programEventDetail/ProgramEventDetailActivity.java

```diff
@@ -14,6 +16,7 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.view.ViewOutlineProvider;
```

Comment:
All these imported classes (Outline, Build, ViewOutlineProvider) are not needed in here.

Reply:
done!